articles/quickstarts/search.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Q# quickstart: TODO"
+description: TODO
+author: cgranade
+ms.author: chgranad@microsoft.com
+ms.date: 10/9/2019
+ms.topic: article
+uid: microsoft.quantum.quickstarts.search
+---
+
+# Search for marked inputs with Grover's Algorithm
+
+This introduction uses [Grover's Algorithm][grover] to search inputs to a function for a particular _marked input_.
+This algorithm can be used to search for inputs with some special property, to find items in an unstructured list, and so forth.
+Grover's algorithm lets us use a quantum computer to run this search in less steps than the number of items in the list that we're searching — something no classical algorithm can do.
+
+In this introduction, we see how we can use Q# and the Quantum Development Kit to run Grover's algorithm with just a few lines of code.
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit][install].
+
+## Writing Grover's Algorithm
+
+Using the Quantum Development Kit, create a new Q# project:
+
+```
+$ dotnet new console -lang Q# --output Grover
+$ cd Grover
+```
+
+Grover's algorithm itself can be written using just a few lines.
+Add the following code to the `Operations.qs` file in your new project:
+
+[!code-qsharp[](~/quantum/samples/algorithms/simple-grover/SimpleGrover.qs?highlight=5,27)]
+
+To define the list that we're searching, create a new file `Reflections.qs`, and paste in the following code:
+
+[!code-qsharp[](~/quantum/samples/algorithms/simple-grover/Reflections.qs)]
+
+The `ReflectAboutMarked` operation in this file represents the problem we're trying to solve with Grover's algorithm, in this case by reflecting bitstings made up of alternating zeros and ones.
```

Comment:
What purpose does reflecting bit strings serve? The goal of this tutorial is to demonstrate impact. Is it possible to use example input that shows a quantum advantage? Or add a couple sentences around why this is interesting? (I recognize that it's a delicate balance around the quantity or information, just wondering if a bit more detail makes sense)

Reply:
There's a couple sentences about that above, as well as a conclusion below that indicates that the right input is found in just four calls; would you recommend adding more detail here?

Reply:
I think what I'm missing is the connection between specifically reflecting bit strings to the search algorithm. After re-reading, I think what might be good is to include before the conclusion some statement like "we are looking for 01010 in a list of n". I only really understood what we were searching the list for after reading the conclusion, so bringing that up would likely make it more clear and provide a nice path to the conclusion. Does that make sense?